src/internal/operations/test/helper_test.go

```diff
@@ -728,6 +728,7 @@ func verifyExtensionData(
 	case path.OneDriveService:
 		detailsSize = itemInfo.OneDrive.Size
 	case path.GroupsService:
+		// FIXME: needs update for message.
```

Comment:
Should we create an issue to track this so that this does not get missed.

Reply:
It gets handled later.